.github/workflows/test.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         node-version:
           - 14.17
           - 16
+          - 17
```

Comment:
good call on adding `17` since that does provide value that we didnt have covered yet. i still think we need `16.0.0` in addition to `16` though, too. `16` will only test against the latest version within the `16` major range, but we're defining in `engines.node` that `16.0.0` is supported, so running verification against that lower bound would help ensure we didnt overlook something

Reply:
actually it may be best to just chang the `16` to `'16.0.0'` since that would be enough to cover the range now that `17` is in the list

Reply:
Ah, I see. When you said `16.0.0`, I figured it was a typo and you meant `17` instead of `16`. 🙃 

OK, fixed.